tests/fake_camera.py

```diff
@@ -0,0 +1,33 @@
+import os
+from typing import Any, Coroutine, List, Tuple, Union
+
+from PIL import Image
+from viam.components.camera import Camera
+from viam.gen.component.camera.v1.camera_pb2 import GetPropertiesResponse
+from viam.media.utils import pil
+from viam.media.video import CameraMimeType, NamedImage, ViamImage
+from viam.proto.common import ResponseMetadata
+
+
+def read_image(name: str):
+    return Image.open(os.path.join("./tests", "img", name + ".jpg"))
```

Comment:
This is probably good enough as-is, but will break if you run the tests from anywhere besides the root of the repo.

If you want to fix it, try replacing `"./tests"` with `os.path.dirname(__file__)`. If you don't want to fix it, I'm fine with leaving it as-is, too.